drivers: wifi: nxp: Fix the compilation error on the MIMXRT1170_EVK board with the NXP_88W8987_MURATA_1ZM_M2 WiFi module

CsBoBoNice opened the pull request:
drivers: wifi: nxp: Fix the compilation error on the MIMXRT1170_EVK board with the NXP_88W8987_MURATA_1ZM_M2 WiFi module

- Fix the compilation error on the MIMXRT1170_EVK board with the NXP_88W8987_MURATA_1ZM_M2 WiFi module

MaochenWang1 commented on drivers/wifi/nxp/nxp_wifi_drv.c:
```diff
@@ -1343,18 +1343,23 @@ static int nxp_wifi_power_save(const struct device *dev, struct wifi_ps_params *
 		case WIFI_PS_PARAM_MODE:
+			
 			if (params->mode == WIFI_PS_MODE_WMM) {
+				#if CONFIG_NXP_WIFI_WMM_UAPSD
```
```suggestion
#ifdef CONFIG_NXP_WIFI_WMM_UAPSD
```

MaochenWang1 replied:
same for other `#if CONFIG_NXP_WIFI_WMM_UAPSD`

CsBoBoNice replied:
This modification was added to prevent compilation errors when the “Disable WMM feature” configuration is applied.

MaochenWang1 commented on samples/net/wifi/shell/boards/mimxrt1170_evk_mimxrt1176_cm7_A.conf:
```diff
@@ -0,0 +1,168 @@
+CONFIG_NXP_WIFI_DRIVER_TASK_STACK_SIZE=4096
+CONFIG_NXP_WIFI_SCAN_TASK_STACK_SIZE=4096
+
+# Disable WMM feature
```
why need this `Disable WMM feature` part?

CsBoBoNice replied:
During the device testing process, I encountered an issue where the device failed to operate normally. My initial assessment suggested that the problem might be due to the CONFIG_NXP_WIFI_WMM configuration being enabled (although later verification revealed this not to be the case). To test this hypothesis, I attempted to disable the configuration, but encountered compilation errors. To resolve this issue, I added the aforementioned macro definitions. After thorough testing, it was confirmed that these new codes do not affect the functionality of the original code, therefore I have retained these modifications.

CsBoBoNice committed:
drivers: wifi: nxp: Fix compilation error with NXP_88W8987_MURATA_1ZM_M2

- Resolve compilation error on MIMXRT1170_EVK board when using the
  NXP_88W8987_MURATA_1ZM_M2 WiFi module.

Signed-off-by: Chen Shu <751541594@qq.com>
